tools/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM quay.io/podman/stable
+
+WORKDIR /project
+
+# Setup Yggdrasil & device-worker repo:
+RUN curl -k https://copr.fedorainfracloud.org/coprs/ondramachacek/yggdrasil/repo/fedora-34/ondramachacek-yggdrasil-fedora-34.repo -o /etc/yum.repos.d/yggd.repo
+RUN curl -k https://copr.fedorainfracloud.org/coprs/eloyocoto/k4e-test/repo/fedora-34/eloyocoto-k4e-test-fedora-34.repo -o /etc/yum.repos.d/flotta-agent.repo && \
+    sed -i 's/\$releasever/34/g' /etc/yum.repos.d/flotta-agent.repo && \
+    sed -i 's/\$basearch/x86_64/g' /etc/yum.repos.d/flotta-agent.repo
+
+# Yum dependencies
+RUN yum install -y golang openssl procps-ng dmidecode flotta-agent nc
```

Comment:
Why golang is needed here?

Reply:
Ah, right. It's leftover because I've previously used the ygg from source not from RPM. Thanks!